### DISCUSSION TITLE
Como conciliar Trabalho + Faculdade + Freelas + Estudos

### DISCUSSION THREAD

**Discussion by hectorgrecco** [frontendbr/forum#1171] [Discussão]
Sim, é até assustador o título da issue, confesso... Mas, acho que de alguma forma alguém vai me dar uma luz, hehe.

Estou num momento de minha vida em que absolutamente TUDO é corrido e eu não tenho tempo pra nada. Minha rotina é basicamente: 

- Trabalhar das 08h-12h30 (CLT)
- Na hora do almoço (13h - 13h50) trabalho nos freelas
- Trabalho mais ainda das 14h-18h
- Das 18h até mais ou menos 23h continuo a trabalhar nos freelas

Enfim, final de semana nem se fala, não sei o que é viver tranquilo num fds há tempos. Além disso tudo, sou músico, tenho umas bandas e realmente não sei como eu arrumo tempo pra fazer shows e outras coisas. Mas, em meio à isso tudo, surge uma dúvida: Como eu consigo conciliar tempo pra isso tudo e ainda estudar (tanto faculdade, que é EAD, quanto estudos pessoais mesmo, seja de desenvolvimento, etc..) ?

O @willianjusten recentemente fez um post muito bacana sobre ser dev e cuidar da sua saúde, porém não faço ideia de onde começar a ter um estilo de vida mais agradável. Isso envolve, ter mais tempo pra estudar minhas coisinhas, fazer uma academia e descansar, pq uma hora o corpo não aguenta (mesmo eu tendo só 19 anos)... Já pensei em parar de fazer freelas, também pensei em largar a faculdade e um bocado de coisas. Realmente, um desespero pra uma pessoa só, hehehe. 

Valeu! =)

**Comment by fxcosta** [comment]
Eu tinha um estilo de vida muito parecido com o seu, com a diferença que aos finais de semana eu ficava com a família / namorada e somente a noite ia estudar / freela.

Te digo de antemão que não é saudável, hoje eu colho alguns frutos disso que é: burnout, stress, sendatarismo e várias outras coisas que parecem pequenas mas que fazem a diferença como: sentir tesão no que faz. O problema é que mesmo não sendo saudável, é o estilo de vida de muita gente e não cabe a mim dizer que é certo ou errado porque isso que fez eu conseguir pagar as contas tantas vezes.

Uma estrategia que eu usava nos freelas era de usar soluções que eu gostaria de estudar. Ex: se aparecia um sistema de chamados pra fazer, eu inventava de fazer com Angular que eu nunca tinha usado. Como freela minha hora era bem cara porque se não, não compensa perder parte da sua vida se for pra ganhar pouco, então já era incluso esse tipo de estratégia. Dessa forma eu me via obrigado a estudar pra implementar a solução e entregar. Deu certo comigo, você precisa analisar se daria certo contigo.

Quanto a faculdade, que obviamente não consegui terminar por causa dessa vida, acaba deixando mais de lado, concluia as materias mas sem brilhantismo e até hoje devo o TCC para conseguir formar.

O que mudou na minha vida? Não muita coisa, hoje sou sócio numa empresa e o tempo que era do freela agora é pra empresa, mas eu me forcei a encontrar tempo para exercícios físicos (por menor que seja, aqueles de 7 minutos mesmo, ta ligado?) e Yoga (tudo em casa mesmo, a ideia é começar de alguma forma).

O mais importante é: seu tempo e sua vida são seus bens mais precisosos. Se você não estiver precisando muito de grana, se não tem que ajudar no sustento da família, enfim, cuide mais da sua saúde, curta mais sua famíllia, use algum projeto bacana seu para estudar uma ferramenta (pode ser algo comercial ou algo que ajude outras pessoas, não importa).

Esse foi um pouco do meu relato, pode não ter ajudado muito, mas sei como é isso porque passei (e passo) por situação semelhante.

**Comment by kivervinicius** [comment]
Mano, por experiência própria e concordando com tudo que o @fxcosta disse, tome muito cuidado com os dramas futuros, tenho muito problema com stress, cabelos brancos, obesidade (mesmo praticando exercicios 4x na semana), ou seja isso te trará no futuro muitos malefícios, uma dica caso poder da uma reduzida nos freelas durante a noite, exemplo, faça uma pausa na quarta feira, e no fim de semana não trabalhe no domingo, fiz isso e me ajudou muito pois você da um tempo pro seu cérebro, evitando um pouco o **burnout**  e o melhor no seu caso eu acredito seria não fazer freelas no almoço, pois vai chegar uma hora que seu cerebro irá parar de processar tão rápido as informações se tornando muito cansativo, você é novo, tente encaixar nesses periodos que eu disse algo que te de prazer, assista uma série, vá ao cinema, fuja de tudo que seja programação ao menos nas folgas, tudo que é demais atrapalha e na nossa área isso é maximizado.

**Comment by hectorgrecco** [comment]
Muito obrigado pelos comentários @fxcosta e @kivervinicius. Realmente, a pior coisa dessa rotina é o stress, e com o stress eu esqueço as coisas muito rápido, o que acaba me prejudicando muito no meu dia a dia e no meu relacionamento com amigos e tal. Tô equilibrando as contas pra justamente poder sair dos freelas e focar nos projetos pessoais. Quando eu tinha namorada ela sofria também em querer sair comigo e eu não ter tempo... hahaha. Vou mudar isso já!

**Comment by ViniciusGularte** [comment]
Tive um estilo de vida muito parecido com o seu, fazia faculdade de engenharia e analise de sistemas ao mesmo tempo e trabalhava ainda... realmente não é saudável, descontava todo stress em comida e acabei engordando uns 20 quilos por conta kk, uma dica como o @fxcosta  e @kivervinicius disseram, pega um dia, faz uma folga, um dia da semana, tenta fugir um pouco da programação, ajuda muito a mente recuperar, essa  rotina vai te prejudicar fisicamente e mentalmente a longo prazo, nosso tempo é nosso maior bem, só podemos dar ele, mas receber não, então tem que usar com cuidado, eu uso muito a técnica pomodore quando estou desenvolvendo, acredito que possa te ajudar já que ela separa blocos pequenos de intervalo para descansar a mente e aumentar a produtividade.

Abraço e sucesso na área.

**Comment by hectorgrecco** [comment]
Valeu, @ViniciusGularteMartin!!! Desde a postagem, não tô mais trabalhando no meu horário de almoço, vou parar de trabalhar nos domingos e vou começar uma academia!!! Meu corpo vai agradecer, hehe

**Comment by dougaraujos** [comment]
@hectorgrecco Conhece conceitos como IKIGAI? Recomendo se aprofundar nisso. Boa parte das nossas preocupações diárias estão atreladas a nossa falta de life planning. Parece meio filosófico e utópico, mas na prática muda tudo.

Links sobre:
https://betterhumans.coach.me/the-japanese-concept-ikigai-is-a-formula-for-happiness-and-meaning-8e497e5afa99

http://www.snap-leadership.com/blog-snap-leadership/2017/3/31/creating-alternate-realities
http://uk.businessinsider.com/no-such-thing-as-one-perfect-life-2017-1

https://www.telegraph.co.uk/health-fitness/mind/finding-ikigai-japanese-secret-health-happiness/
